### ISSUE TITLE
vagrant package --include :: Try the Include First

### ISSUE THREAD

**Issue by lloy0076** [hashicorp/vagrant#3199]
TL;DR

It seems that the files included get included last; I wonder if it would be faster to do the include (somehow) first or at least figure out that Ruby won't full over before exporting the VM.

Theoretically, the export of the VM will take longer than the --include BUT if the --include fails for whatever wonky reason, it's a waste of time.

LONG

I keep getting:

```
vagrant package --base 'Arkade Centos 64 Take II' --include DEPLOY-SCRIPTS-final --vagrantfile ./DEPLOY-SCRIPTS-final/ --out Arkade-Centos-64-Deploy.box
[Arkade Centos 64 Take II] Clearing any previously set forwarded ports...
[Arkade Centos 64 Take II] Exporting VM...
[Arkade Centos 64 Take II] Compressing package to: /Users/lloy0076/Documents/work/agix/arkade/PAS/provision/vagrant-deploy/Arkade-Centos-64-Deploy.box
[Arkade Centos 64 Take II] Packaging additional file: DEPLOY-SCRIPTS-final
[Arkade Centos 64 Take II] Packaging additional file: ./DEPLOY-SCRIPTS-final/
/Applications/Vagrant/embedded/lib/ruby/2.0.0/fileutils.rb:1353:in `symlink': File exists - (misc/hiera, /Users/lloy0076/.vagrant.d/tmp/1394757261/include/DEPLOY-SCRIPTS-final/hiera) (Errno::EEXIST)
...
```

This is not what I complaining about.

Before this, Vagrant does a VirtualBox export which takes, say, a minute; and then summarily crashes.

Obviously this is only in my case but I suspect the VM export will take longer than packaging the include. Or, on multi-threaded machines, they could run ... in parallel.

Is it sensible to see whether the --include works (by doing it?) then doing the Vagrant VM export?
